packages/cfd/src/Containers/cfd-password-manager-modal.tsx

```diff
@@ -53,6 +53,8 @@ const CountdownComponent = ({ count_from = 60, onTimeout }: TCountdownComponent)
 
         onTimeout();
 
+        // eslint-disable-next-line @typescript-eslint/no-empty-function
+        return () => {};
```

Comment:
I saw a `clearTimeout` within the `count` check block. I think we can move it here instead of putting an empty cleanup function